### PR TITLE
added math.factorial() page

### LINKED ISSUE #5032: [Term Entry] Python math-module math.factorial()
### Reason/inspiration (optional)

We would like a new term entry on the `math.factorial()` method of the [`math-module`](https://www.codecademy.com/resources/docs/python/math-module) concept in Python. The entry should go in a new file under `docs/content/python/concepts/math-module/terms/math-factorial/math-factorial.md`

The entry should include the following:

- A description of the term
- A ##Syntax section that includes the syntax of the method and its details
- An ##Example section that includes a ```py ``` codeblock and ```shell ``` that shows the example output.
- A ##Codebyte Example to show the working of the method

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] By submitting this issue, I agree to follow Codecademy Doc's [Code of Conduct](https://github.com/Codecademy/docs/blob/main/.github/CODE_OF_CONDUCT.md).

### For Maintainers

- [X] Labels added
- [ ] Issue is assigned

### PR DESCRIPTION
**Pull request by yuekaim**
### Description

added math.factorial() page at python/concepts/math-module/terms

### Issue Solved
Closes #5032

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Review by cigar-galaxy82 — CHANGES_REQUESTED** [review]

**Comment by cigar-galaxy82** [comment]
Hi @yuekaim thank you for contributing to Codecademy docs, I have suggested a few suggestions please modify

**Comment by yuekaim** [comment]
@cigar-galaxy82 Thanks for the feedback. Sorry for the delay. I have edited to reflect them. I am currently experiencing some local node issues that's taking a while to resolve. I'll push as soon as that's fixed. Thank you.

File: content/python/concepts/math-module/terms/math-factorial/math-factorial.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  - 'Data Science'
+Tags:
+  - 'Arithmetic'
+  - 'Functions'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The `math.factorial()` method returns the factorial of a positive number.
```
Comment (SaviDahegaonkar) [inline content/python/concepts/math-module/terms/math-factorial/math-factorial.md:15]:
```suggestion
The `math.factorial()` method returns the factorial of a positive number.
```
The first mention of the concept must always be in **bold**. Please make this correction here.

**Review by SaviDahegaonkar — CHANGES_REQUESTED** [review]
Hey Author,
The entry is nicely written. Please implement the changes I requested from you so that I can approve and merge them.

Thanks,
Savi

**Review by SaviDahegaonkar — APPROVED** [review]
The entry LGTM!

Thanks,
Savi

**Review by cigar-galaxy82 — APPROVED** [review]

**Pull request merged**